cmd/bootstrap/cmd/partner_infos.go

```diff
@@ -0,0 +1,265 @@
+package cmd
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-core-contracts/lib/go/templates"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go-sdk/client"
+	"github.com/onflow/flow-go/cmd"
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/encodable"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	flagOutputDir    string
+	flagANAddress    string
+	flagANNetworkKey string
+	flagNetworkEnv   string
+
+	getNodeInfoScript = []byte(`import FlowIDTableStaking from 0x9eca2b38b18b5dfe
+
+	// This script gets all the info about a node and returns it
+	
+	pub fun main(nodeID: String): FlowIDTableStaking.NodeInfo {
+		return FlowIDTableStaking.NodeInfo(nodeID: nodeID)
+	}`)
+)
+
+const (
+	// Index of each field in the cadence NodeInfo as it corresponds to cadence.Struct.Fields needed to build NodePubInfo struct,
+	// fields not needed are left out.
+	idField = iota
+	roleField
+	networkingAddressField
+	networkingKeyField
+	stakingKeyField
+	tokensStakedField
+	flowNodeAddrPart    = "nodes.onflow.org"
```

Comment:
```suggestion
	tokensStakedField
)

const (
	flowNodeAddrPart    = "nodes.onflow.org"
```
Bit of a nitpick, but I think it's clearer to keep the `iota`-based declarations in their own `const` block.

Reply:
https://github.com/onflow/flow-go/pull/1988/commits/cba109821575449a48fc29c352bf60649e5a3249